tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -15,6 +16,14 @@ def abs_path(path):
     return os.path.abspath(os.path.expanduser(path))
 
 
+def cpu_count():
+    cpu_count = multiprocessing.cpu_count()
```

Comment:
I broadly agree with scaling the default `processes` linearly with CPU count, but I think we need to cut this by a constant factor >= 2 for optimal usage; otherwise, there could be too much contention. Each runner manager spawns at least two processes (the runner and browser). For Chromium, there are also more processes under the browser.

Reply:
Did a test in chromium, and updated it here to make it available to all reviewers. When running with half number of workers (4 from previous 8), the total test time increased by about 15%. I guess we don't fully understand the interactions in the system. One hypothesis is that running with less workers makes the cpu less saturated, which outweighs the gain due to less context switch.